e2-studio-star-rx72n-firmware/libs/rx_hcsr04/src/rx_hcsr04_icu.c

```
@@ -0,0 +1,254 @@
+ * @post IPR[vector] contains priority
+ * @post IER[vector/8] bit set (interrupt enabled)
+ *
+ * @note Not thread-safe. Call during initialization only.
+ * @warning Do not call while a measurement is in progress.
+ *
+ * @see rx_mpc_set_irq() Configure pin before calling this
+ * @see rx_hcsr04_icu_disable() Reverse this configuration
+ *
+ * @since Version 1.2.0 (Issue #296)
+ */
+rx_err_t rx_hcsr04_icu_configure(const uint8_t irq_num, const uint8_t priority)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_hcsr04_icu_configure' of similar type ('const uint8_t') are easily swapped by mistake

(bugprone-easily-swappable-parameters)